tests/neg/custom-show.scala

```diff
@@ -0,0 +1,20 @@
+import scala.compiletime.{CustomShow, CustomShowable}
+object Regular:
+  trait Foo[T]
+  type Baz = Foo["To Infinity And Beyond!"]
+  val x: Foo["To Infinity And Beyond!"] = 1 // error
```

Comment:
You can add a `custom-show.check` file in the same directory, to check the actual error messages that are produced. Otherwise this test does not really test the introduced feature.